src/main/scala/com/cloudera/sparkts/Lag.scala

```diff
@@ -97,4 +97,28 @@ private[sparkts] object Lag {
       }
     }
   }
+
+  /**
+   * Creates a lagged matrix from a current matrix (represented in row-array form). Lags each column
+   * the appropriate amount of times and then concatenates the columns.
+   * So given a matrix [a b c], where a/b/c are column vectors, and calling with lag of 2, becomes a
+   * matrix of the form [a_-1 a_-2 b_-1 b_-2 c_-1 c_-2]
+   */
+  def lagMatTrimBoth(
+                      x: Array[Array[Double]],
```

Comment:
Spark-TS style convention is to indent method arguments four spaces past the beginning of the `def`